rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/FailedLRAIT.java

```diff
@@ -94,15 +96,19 @@ public void after() {
         }
     }
 
+    static String getRecoveryUrl(URI lraId) {
+        return LRAConstants.getLRACoordinatorUrl(lraId) + "/" + RECOVERY_COORDINATOR_PATH_NAME;
+    }
+
     /**
      * test that a failure record is created when a participant (with status reporting) fails to compensate
      */
     @Test
     public void testWithStatusCompensateFailed() throws Exception {
-        URI lra = invokeInTransaction(LRAParticipantWithStatusURI.LRA_PARTICIPANT_PATH,
+        LRAURIHolder lra = invokeInTransaction(LRAParticipantWithStatusURI.LRA_PARTICIPANT_PATH,
```

Comment:
Can we stay with using URIs since LRAURIHolder only adds overhead on top of that.